src/components/Heading/Heading.stories.module.css

```diff
@@ -28,5 +28,7 @@
 
 .variant--dark {
   grid-column: 3;
-  background-color: var(--eds-theme-color-background-neutral-default-inverted);
+  background-color: var(
+    --eds-theme-color-background-neutral-default-inverted
+  ); /** AH-Strip? */
```

Comment:
I think the usage of the term `inverted` in tokens and mixins and such is fine. I just don't want the prop options when we haven't actually worked those styles out with design and don't intend for consumers to utilize them.

Reply:
@dierat Sounds good. "inverted" might be a smidge overloaded, so I flagged a few other places like that (which I think are all in the "leave alone" category)